packages/cos/collection.yaml

```diff
@@ -1,25 +1,25 @@
 packages:
   - name: "cos"
     category: "system"
-    version: "0.5.7+5"
```

Comment:
I'd rather the minor version as there is a change in GRUB which wouldn't be got in from the old installs (unless they do a cos-reset which re-applies grub changes)